Fix #2: Úprava deploy skriptu pro automatické pravidelné generování datasetu

michto01 opened the pull request:
Upravený deploy skript tak, aby každou sobotu v 23:59 přegeneroval dataset.

michto01 committed:
Fix #2: Modify deploy workflow to be automatically sheduled.

Modify the deploy workflow script to re-generate the output
every Saturday at 23:59.

zoul commented on .github/workflows/deploy.yml:
```diff
@@ -3,6 +3,8 @@ name: Deploy
 on:
   push:
     branches: [master]
+  schedule:
```
Víme, jak tohle interaguje s větvemi? Deploy se udělá vždycky po pushnutí do masteru, to je jasný, ale ten naplánovaný trigger jede nad kterou větví? V dokumentaci píšou:

> Scheduled workflows run on the latest commit on the default or base branch.

Což by byla větev `next`? Neměli bysme v té akci `checkout` níže nastavit radši explicitně `ref` na `master`? (Viz [dokumentaci pro checkout](https://github.com/actions/checkout).)

michto01 replied:
Chybička se vloudila, měl by řešit  [bde77d6](https://github.com/cesko-digital/obce/pull/24/commits/bde77d6de29a85d20df2c8afff54f72d089df9d2)

michto01 committed:
Add master as designated target for Deployment

Add `master` as  the branch to be deployed against instead of the default `next` in cast of schedule cron

zoul commented:
Bezvadný, díky. V neděli uvidíme :)
